conf/application.conf

```diff
@@ -106,10 +106,10 @@ microservice {
         protocol = http
         host = localhost
         port = 9570
-        max-uploads = 5
+        max-uploads = 4
         min-file-size = 0
         max-file-size = 5242880
-        upscan-store.expiry-time = 7 days
+        upscan-store-expiry-time = 7
```

Comment:
having the units either on the left or the right of the equals sign would be better than no units at all

Reply:
will address after 6th April